app/gist/template.hbs

```diff
@@ -34,10 +42,14 @@
       }}
     </ul>
 
-    <div class="title">
-      {{title-input value=model.description titleChanged=(action "titleChanged")}}
-      {{saved-state-indicator model=model unsaved=unsaved}}
-    </div>
+    {{#if (or media.isDesktop media.isJumbo)}}
+      <div class="title">
+        {{title-input value=model.description titleChanged=(action "titleChanged")}}
+        {{saved-state-indicator model=model unsaved=unsaved}}
+      </div>
```

Comment:
Title was changed before this PR, so now it blocks clicking on the navlinks when above the collapse threshold an below ~1500px, see canary.ember-twiddle.com